src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private int index = new Random().nextInt(Colors.values().length);
```

Comment:
In the current implementation index will always be the same, class level field should be only new Random(), like you did in Lottery class